internal/http/services/owncloud/ocdav/move.go

```diff
@@ -99,7 +100,9 @@ func (s *svc) handleSpacesMove(w http.ResponseWriter, r *http.Request, srcSpaceI
 	ctx, span := rtrace.Provider.Tracer("ocdav").Start(r.Context(), "spaces_move")
 	defer span.End()
 
-	dst, err := extractDestination(r)
+	dh := r.Header.Get(net.HeaderDestination)
+	baseURI := r.Context().Value(net.CtxKeyBaseURI).(string)
+	dst, err := net.ParseDestination(baseURI, dh)
```

Comment:
Is there a reason why `extractDestination` was removed? I mean the three lines above could also be in that function, then no code changes would be needed here.

Reply:
`extractDestination` was moved to `net.ParseDestination`. It just doesn't get the value from the header itself because this way the parse function is more generic and you can have different baseURIs.